misc/deployment/BuyerWeb/azure-deployment-build-pipeline.yml

```diff
@@ -0,0 +1,55 @@
+trigger:
```

Comment:
**suggestion:** it should build and push both buyer-web and buyer-web-react-ssr  apps

Reply:
Okay, I will try today

Reply:
Should I add it in the same task or another?